components/input/input.css

```diff
@@ -0,0 +1,73 @@
+.form-label {
+    display: block;
+    margin-top: 1rem;
+    margin-right: 1rem;
+}
+
+.form-field {
+    display: block;
+    width: 100%;
+    padding: 0.5rem 0.5rem;
+    border: 1px solid var(--sleek-gray-color-400);
+    border-radius: var(--border-radius-extra-small);
+    margin: 0.5rem 0
+}
+
+.form-field:focus {
+    border: 1px solid var(--info-color);
+    outline: 0.25rem solid rgba(144, 205, 244, 0.3);
```

Comment:
My personal choice, for **reusability,** I even make variables for shadow color and outline color. Example,
--outline-color: rgba(144, 205, 244, 0.3) ;

Reply:
Yes Ruchi, I do it the same way. But as I'm using a separate branch for the variables, I'll have to switch to it, so I decided to hard code the values for now.